src/domain/signupGroup/signupGroupForm/signups/signup/Signup.tsx

```diff
@@ -83,6 +86,9 @@ const Signup: React.FC<SignupProps> = ({
     t,
   ]);
 
+  const getSplittedRowType = (type: SplittedRowType): SplittedRowType =>
+    readOnly ? 'medium-medium' : type;
```

Comment:
Maybe the splitted row types like `medium-medium` could be defined as constants in a constants file to ensure a single source of truth?

Reply:
Added `SPLITTED_ROW_TYPE` enum and `DEFAULT_SPLITTED_ROW_TYPE` constant to define type for the SplittedRow component